### PR TITLE
Preparing release v1.33.0

### LINKED ISSUE #1851: Release 1.33.0
Jaeger core 1.33.0 has been released https://github.com/jaegertracing/jaeger/releases/tag/v1.33.0. Now the operator should be released.

### PR DESCRIPTION
**Pull request by rubenvp8510**
Signed-off-by: Ruben Vargas <ruben.vp8510@gmail.com>

## Which problem is this PR solving?
- Fixes #1851 

## Short description of the changes
-  Pepare release 1.33.0, changelog, manifest etc..

### CODE REVIEW

File: COMPATIBILITY.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+The following table shows the compatibility of jaeger operator with different components, in this particular case we shows Kubernetes and Strimzi operator compatibility
+
+
+| Jaeger Operator | Kubernetes           | Strimzi Operator   |
```
Comment (frzifus) [inline COMPATIBILITY.md:4]: Should we then replace this table with a link?
https://github.com/jaegertracing/jaeger-operator#compatibility-matrix

File: config/manifests/bases/jaeger-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -124,7 +124,7 @@ spec:
   maturity: alpha
   provider:
     name: CNCF
-  replaces: jaeger-operator.v1.31.0
+  replaces: jaeger-operator.v1.30.0
```
Comment (frzifus) [inline config/manifests/bases/jaeger-operator.clusterserviceversion.yaml:127]:
```suggestion
  replaces: jaeger-operator.v1.32.0
```
Does it not replace 1.32?

File: bundle/manifests/jaeger-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -518,11 +518,11 @@ spec:
   maturity: alpha
   provider:
     name: CNCF
-  replaces: jaeger-operator.v1.31.0
+  replaces: jaeger-operator.v1.30.0
```
Comment (frzifus) [inline bundle/manifests/jaeger-operator.clusterserviceversion.yaml:521]: same here

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,18 @@
 Changes by Version
 ==================
 
+1.32.0 (2022-04-14)
```
Comment (frzifus) [inline CHANGELOG.md:4]:
```suggestion
1.33.0 (2022-04-14)
```
Oh and that one ^^

**Comment by frzifus** [comment]
Is this missing? https://github.com/jaegertracing/jaeger-operator/pull/1732

File: COMPATIBILITY.md (hunk 1 above)
Comment (rubenvp8510) [reply COMPATIBILITY.md:4]: We can do this in a separate PR, I would prefer to use this PR only for the release.

**Comment by rubenvp8510** [comment]
> Is this missing? #1732

Not really, it wasn't merged when I created this PR :)

File: COMPATIBILITY.md (hunk 1 above)
Comment (frzifus) [reply COMPATIBILITY.md:4]: Should we then update the Readme too? Thats the first thing people see when they go on the github page.

**Review by frzifus — APPROVED** [review]
LGTM :)

**Pull request merged**